src/org/exoplatform/widget/MyActionBar.java

```diff
@@ -1,275 +1,276 @@
-                                              View.VISIBLE);
-    getActionBar().setVisibility(visibility);
-  }
-
-  @Override
-  public void setTitle(CharSequence title) {
-    getActionBar().setTitle(title);
-  }
-
-  @Override
-  public void setTitle(int titleId) {
-    setTitle(getString(titleId));
-  }
-
-  public ActionBar getActionBar() {
-    ensureLayout();
-    return mActionBarHost.getActionBar();
-  }
-
-  public ActionBarItem addActionBarItem(ActionBarItem item) {
-    return getActionBar().addItem(item);
-  }
-
-  public ActionBarItem addActionBarItem() {
-
-    ActionBarItem item = getActionBar().newActionBarItem(NormalActionBarItem.class);
-
-    return getActionBar().addItem(item);
-
-  }
-
-  public ActionBarItem addActionBarItem(int itemId) {
-
-    @SuppressWarnings("deprecation")
-    final Drawable d = new ActionBarDrawable(this.getResources(), itemId);
-
-    ActionBarItem item = getActionBar().newActionBarItem(NormalActionBarItem.class).setDrawable(d);
-
-    return getActionBar().addItem(item);
-
-  }
-
-  public ActionBarItem addActionBarItem(ActionBarItem item, int itemId) {
-    return getActionBar().addItem(item, itemId);
-  }
-
-  public ActionBarItem addActionBarItem(ActionBarItem.Type actionBarItemType) {
-    return getActionBar().addItem(actionBarItemType);
-  }
-
-  public ActionBarItem addActionBarItem(ActionBarItem.Type actionBarItemType, int itemId) {
-
-    return getActionBar().addItem(actionBarItemType, itemId);
-  }
-  
-
-  public FrameLayout getContentView() {
-    ensureLayout();
-    return mActionBarHost.getContentView();
-  }
-
-  public void setActionBarContentView(int resID) {
-    LayoutInflater.from(this).inflate(resID, getContentView());
-    Drawable bg = getActionBar().getBackground();
-    if (bg != null && (bg instanceof BitmapDrawable)) {
-      BitmapDrawable bmDrawable = (BitmapDrawable) bg;
-      bmDrawable.setTileModeX(TileMode.REPEAT);
-      getActionBar().setBackgroundDrawable(bmDrawable);
-    }
-  }
-
-  public void setActionBarContentView(View view, LayoutParams params) {
-    getContentView().addView(view, params);
-  }
-
-  public void setActionBarContentView(View view) {
-    getContentView().addView(view);
-  }
-
-  public boolean onHandleActionBarItemClick(ActionBarItem item, int position) {
-
-    return true;
-  }
-
-  private OnActionBarListener mActionBarListener = new OnActionBarListener() {
-                                                   public void onActionBarItemClicked(int position) {
-
-                                                     onHandleActionBarItemClick(getActionBar().getItem(position),
-                                                                                position);
-
-                                                   }
-                                                 };
-
-  public GDApplication getGDApplication() {
-    return (GDApplication) getApplication();
-  }
-
-}
+// // TODO delete
```

Comment:
The only benefit of keeping and using this empty parent is if later on we need to provide some common behavior for all activities. Currently we don't. Do you have any idea ?

About using ToolBar, it is nice indeed, I'll try it even though I have to add it in each layout...
FYI http://android-developers.blogspot.com/2014/10/appcompat-v21-material-design-for-pre.html